COMMUNITY-ROLES.md

```diff
@@ -191,14 +189,14 @@ using one of the template messages below as a base.
    go to https://github.com/orgs/tldr-pages/people, click the gear icon in their row,
    and select the "Convert to outside collaborator" menu entry.
 
-3. Open a PR moving their name to the "Past organization owners" section
-   in [MAINTAINERS.md](MAINTAINERS.md).
-   Make sure to include `Closes #<issue number>` in the PR description.
+3. Open a PR moving their name to the end of the "Organization owners" section
+   in [MAINTAINERS.md](MAINTAINERS.md#organization-owners).
+   **Make sure to include `Closes #<issue number>` in the PR description.**
    The issue will then be automatically closed once the PR is merged.
 
 
 ## Who can change roles
 Any member of the community can (and is encouraged to) propose role changes
-by following the process outlined [above](#how-to-change-roles).
+by following the process outlined above.
```

Comment:
I removed the link here because it confused me for a second, because the only the thing that separates it and the next link is the period at the end of the sentence.  It doesn't seem particularly useful to link to the immediately succeeding thing, either.

Reply:
I'd keep it anyway. It'll render nicely in markdown right? It just aids clarity and it doesn't hurt anything I guess

Reply:
I agree with @sbrl, this is helpful in navigating the rendere document, which isn't exactly short, and it's separated from the next line by a full stop, so it shouldn't be too troublesome.

Reply:
Also, not sure why you marked this as resolved since from what I can see, the diff still removes the link 🤔. Unresolving for the time being, for visibility.